lib/pronto/formatter/github_pull_request_formatter.rb

```diff
@@ -31,6 +31,20 @@ def create_comment(client, sha, body, path, position)
         # Ignore such occasions and continue posting other messages.
         STDERR.puts "Failed to post: #{comment.inspect} with #{e.message}"
       end
+
+      def duplicate_create_comment(client, sha, body, path, position)
```

Comment:
Similar code found in github_pull_request_formatter.rb:21, github_pull_request_formatter.rb:35